### PR TITLE
2865: Fix not found error flickering for clicking on push notification

### PR DESCRIPTION
**Pull request by steffenkleinle**
### Short description

Fix not found error flickering for clicking on push notification.

### Proposed changes

- Force refresh for initial local news load in NewsContainer
- Remove effect to refresh which caused a race condition

### Side effects

None.

### Resolved issues

Fixes: #2865.

---

### CODE REVIEW

File: native/src/hooks/useLoadCityContent.ts [hunk 1]
```diff
@@ -27,6 +27,7 @@ import useSnackbar from './useSnackbar'
 type Params = {
   cityCode: string
   languageCode: string
+  refreshLocalNews?: boolean
```
Comment (LeandraH) [inline native/src/hooks/useLoadCityContent.ts:30]: 🔧  Can we call this something like `refreshForLocalNews`? `refreshLocalNews` sounds to me like it should be a function.

**Review by LeandraH — COMMENTED** [review]
Unfortunately, I get the flickering on this branch on my iPhone :(

**Comment by steffenkleinle** [comment]
> Unfortunately, I get the flickering on this branch on my iPhone :(

Are you sure you are on the right branch? Can you record this? This sadly does not give me any information on where to start. :see_no_evil:

**Comment by LeandraH** [comment]
Just rechecked, definitely on the branch "2865-not-found-flickering". I do seem to remember this only being a problem with the test CMS though, and that the production CMS is fast enough?

https://github.com/user-attachments/assets/6e59e11b-1493-42c2-a11e-bca191a9406f

**Comment by steffenkleinle** [comment]
> Just rechecked, definitely on the branch "2865-not-found-flickering". I do seem to remember this only being a problem with the test CMS though, and that the production CMS is fast enough?

@LeandraH thanks for the video, but it seems like you are testing different stuff than what the issue is about? The issue is about clicking a PN and then shortly seeing a not found error.
In your video, you are not doing that and only a `no languages available` error is displayed :see_no_evil: Am I missing something?

**Comment by LeandraH** [comment]
Whooops, indeed, wrong video, sorry!


https://github.com/user-attachments/assets/59ec717e-bce8-4500-a4bb-7e37f3e68c3b

**Review by LeandraH — APPROVED** [review]
Sorry, this one also works once the hot-reloading works 🤦‍♀️  Tested on real iOS.

**Review by f1sh1918 — APPROVED** [review]
Tested on android. works fine

**Pull request merged**